src/components/DeleteButton.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+import icon_bin from '../images/icon_bin.svg';
+
+export default function DeleteButton({ handleDelete, id }) {
+  return (
+    <DeleteButtonStyled type="button" onClick={handleDelete} id={id}>
+      <img src={icon_bin} alt="bin icon" width="13px" height="16px" />
```

Comment:
This is a valid option to  create this button. However you could us a more general button component and pass down the img as a child.

Reply:
thanks for this hint - indeed i was thinking about it, but chose this option as i have not planned another button of this format so far - i will keep in mind to change it, when i should add similar buttons